vpn-ca/main.go

```diff
@@ -185,11 +202,11 @@ func main() {
 		os.Exit(1)
 	}
 
-	caInfo := getCa(*caDir)
+	caInfo := getCa(caDir)
 
 	if "" != *serverCommonName {
 		validateCommonName(*serverCommonName)
-		sign(caInfo, *serverCommonName, getServerTemplate(*serverCommonName, &caInfo.caCert.NotAfter))
+		sign(caInfo, *serverCommonName, getServerTemplate(*serverCommonName, &caInfo.caCert.NotAfter), serverSign)
```

Comment:
maybe you can use something like "targetDir" instead of "serverSign"... that would be more clear in what it means...

Reply:
has been changed in the latest commit to targetDir